lisa/analysis/rta.py

```diff
@@ -198,6 +198,64 @@ def save_plot(self, figure, filepath=None, img_format=None):
         default_dir = dirnames.pop()
         return self._save_plot(figure, default_dir, filepath, img_format)
 
+    @memoized
+    def df_activations(self, task, start_time=None):
```

Comment:
Is there a fundamental difference with what the `tasks.py` analysis can provide ?
For reference, this PR provides a method that seems similar, but built on top of the trace.dat, which sounds more generally re-usable than rt-app specific data:
https://github.com/ARM-software/lisa/pull/915/commits/af995ff9a85d571fc95935a6cc56f5e25de96572